README.md

```diff
@@ -13,12 +13,14 @@ pipenv shell
 
 ## Running It:
 
-To run it you'll need to set the following environment variables, either directly, or using Foreman or Envdir:
+To run it you'll need to set the following environment variables. This can be done by running `source config.env` or using Foreman or Envdir:
```

Comment:
Please could you make it clearer in this sentence that they need to __modify__ config.env before sourcing it?